package.json

```diff
@@ -30,25 +32,56 @@
         "react-router": "^5.0.1",
         "react-router-dom": "^5.0.1",
         "redux": "^4.0.1",
-        "redux-saga": "^1.0.5"
+        "redux-saga": "^1.0.5",
+        "sinon": "^7.3.2"
```

Comment:
`sinon` is dev depth, don't think you really need this, mostly everything could be done with `jest`

Reply:
Followed documentation